challenges/prototypes.js

```diff
@@ -1,33 +1,55 @@
 /* ===== Prototype Practice ===== */
 
+console.log("Prototype Challenges 👇")
+
 // Task: You are to build a cuboid maker that can return values for a cuboid's volume or surface area. Cuboids are similar to cubes but do not have even sides. Follow the steps in order to accomplish this challenge.
 
 /* == Step 1: Base Constructor ==
   Create a constructor function named CuboidMaker that accepts properties for length, width, and height
 */
 
+function CuboidMaker(attributes) {
+  this.length = attributes.length;
+  this.width = attributes.width;
+  this.height = attributes.height;
+};
```

Comment:
Good work on this, there are other ways to go about it as well, but this actually works for prototypes.